src/custom.ts

```diff
@@ -21,6 +21,10 @@ class CustomType<InputType, OutputType> extends BaseType<InputType, OutputType,
   is(value: any): value is this["InputType"] | this["InstanceType"] {
     return this.mstType.is(value);
   }
+
+  async schemaHash() {
+    return `custom`;
```

Comment:
Don't we need to incorporate `options` into the hash? Not all custom types are equivalent? (That's an assumption I'm not entirely sure what these look like).

Custom types also may be unique? Maybe we could generate a unique ID for them and re-use that?

Reply:
Good call, updated to make custom types use a unique id